code/datums/diseases/advance/symptoms/damage_converter.dm

```diff
@@ -42,6 +42,10 @@ Bonus
 
 		var/list/parts = H.get_damaged_organs(1,1) //1,1 because it needs inputs.
```

Comment:
Adding a new argument does not require you to update all old calls, it's just good practice.